_data/banking.yml

```diff
@@ -330,6 +330,14 @@ websites:
       software: Yes
       doc: https://www.firsttechfed.com/OnlineServices/bankonlinesecurely/SecurityKey
 
+    - name: FirstBank (Colorado)
```

Comment:
i think for how this site tends to be organized, those would be named `First Banks`, `First Bank & Trust Company`, `First Bank of Berne`, etc. those sites can be dealt with when submitted. thanks for the changes!